### DISCUSSION TITLE
How to change existing mysql-database-settings from latin1_swedish_ci to utf8mb4?

### DISCUSSION THREAD

**Discussion by WeT-Klb** [nocodb/nocodb#1411] [General]
In [https://github.com/nocodb/nocodb/issues/1313](url) there´s a notice to change settings for mysql from latin1_swedish_ci to utf8mb4?
If the database already exists - how to change then?
Or will the new docker file change it automatically?

**Comment by wingkwong** [comment]
If you have access to your DB, you can run the below SQL. 

```sql
ALTER DATABASE
    db_name # TODO: change db_name
    CHARACTER SET = utf8mb4
    COLLATE = utf8mb4_unicode_ci;
```

```sql
ALTER TABLE
    table_name # TODO: change table_name
    CONVERT TO CHARACTER SET utf8mb4
    COLLATE utf8mb4_unicode_ci;
```
